packages/react/src/reactrouterv6.tsx

```diff
@@ -94,13 +108,31 @@ function getNormalizedName(
 
   const branches = matchRoutes(routes, location);
 
+  let pathBuilder = '';
   if (branches) {
     // eslint-disable-next-line @typescript-eslint/prefer-for-of
     for (let x = 0; x < branches.length; x++) {
```

Comment:
Just out of curiosity: why the index for loop?

Reply:
It's to stay consistent with RR5 integration. Not sure if there's any specific reason there though.

https://github.com/getsentry/sentry-javascript/blob/9b7131fcbf7f47be3a78893fa56a70da6c29ee7a/packages/react/src/reactrouter.tsx#L79-L84

Reply:
I'll just keep the explicit for loop, though technically it is extra bytes